### PR TITLE
Flow board begin date

### PR DESCRIPTION
**Pull request by sloboy**
Give option for flow board begin date: Same day or beginning of work week.

Also added option for flow board end date: D7 one week ahead.

Not the most elegant selection structure, but I am young in php so I don't really know all the date math functions yet.

### CODE REVIEW

**Comment by MatthewVita** [comment]
Hi @sloboy,

Looks good so far. Can you check if things like `strtotime('previous monday')` will work in our supported PHP versions?

**Comment by MatthewVita** [comment]
@sloboy okay, I thank you for looking into that. PHP does indeed appear to support this across the relevant versions!

The code looks great. Can you just give it a quick smoke test on your end and then we can merge? I normally like to help with the QA bits of reviews, but I'm afraid I'm short on time this week :(

File: library/globals.inc.php [hunk 1]
```diff
@@ -1660,6 +1661,16 @@ function gblTimeZones()
...
         ),
 
+        'ptkr_start_date'=> array(
+            xl('Flow Board: Default Starting Date'),
+            array(
+                'D0' => xl('Current Day'),
+                'B0' => xl('Beginning of Work Week (First day in week above)'),
+            ),
+            'B0',                    // default = Current Day
+            xl('This is the default Beginning date for the Patient Flow Board.  Use Current Day if Allow Date Range is set to no.')
```
Comment (bradymiller) [inline library/globals.inc.php]: Don't include double spacing in translation constants.

**Comment by bradymiller** [comment]
@sloboy and @MatthewVita ,
Agree this code looks excellent!
@slobog, just let is know it is testing well, and will bring it in (@MatthewVita , after bringing this in, we'll need to do a psr2 run on it to make it psr2 compliant and increment database counter in version.php, which I'm happy to do; @sloboy, for the future, rec trying to get your editor to support PSR2; not mandatory since it's easy for us to do this, but is good practice to start doing this).

**Comment by bradymiller** [comment]
hi @sloboy ,
I just brought in your feature into the codebase. Thanks for this awesome contribution!
Note I made some minor changes (and also fixed a previous bug where when the Flow Board: Allow Date Range is turned to off, the start and end date are basically current day):
https://github.com/openemr/openemr/commit/fc8d5968ebc41e117abf0799c2377b6d6ed51275
https://github.com/openemr/openemr/commit/cc367480a4e11abc982053f191dc0ae47e57fd42

**Comment by sloboy** [comment]
Thank you!  @MatthewVita @bradymiller 

I am learning how to get a PSR2 checker and fixer into my environment (windows).  I did not have a wamp setup locally, but was just developing into the docker container.

**Pull request merged**